uint8 BASE_LINK_AS_REFERENCE=0
uint8 FOOT_LINK_AS_REFERENCE=1
uint8 HIP_LINK_AS_REFERENCE=2

uint8 reference_link
geometry_msgs/Vector3 body_position
geometry_msgs/Vector3 body_rotation
black_mouth_kinematics/AllLegPoints leg_points